internal/xds/translator/testdata/out/xds-ir/http3.routes.yaml

```diff
@@ -0,0 +1,17 @@
+- ignorePortInHostMatching: true
+  name: envoy-gateway/gateway-1/tls
+  virtualHosts:
+  - domains:
+    - '*'
+    name: envoy-gateway/gateway-1/tls/*
+    routes:
+    - match:
+        prefix: /
+      name: httproute/default/httproute-1/rule/0/match/0/*
+      responseHeadersToAdd:
+      - append: true
+        header:
+          key: alt-svc
+          value: h3=":10443"; ma=86400
```

Comment:
This port needs to be the port exposed externally outside the cluster to the client. So, in most cases involving a browser it is more than likely going to be `:443` and not the listener port. The LoadBalancer of type `Service` then needs to point traffic to 10443. That is what I was referring to as the tricky part because a single Service doesn't always allow for MixedProtocol. _note: that might have change since last time I looked into this but it is the more nuanced part to the setup._